crates/ironrdp-web/src/session.rs

```diff
@@ -729,13 +740,14 @@ impl Session {
         Ok(())
     }
 
+    // QUESTION: in VNC we cannot synchronize lock keys, so do we need to remove this from the Session API?
```

Comment:
How should we approach it? Should it be part of the extensions API?

Reply:
If it’s not part of the common API, yes, we should turn that into an extension! :+1:

Reply:
Done